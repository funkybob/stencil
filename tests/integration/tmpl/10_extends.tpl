{% extends "10_parent.html" %}
{% block title %}Welcome!{% endblock %}
content outside blocks are ignored
{% block footer %}Footer{% endblock %}
