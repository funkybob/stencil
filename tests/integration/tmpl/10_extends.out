<title>Welcome!</title>
<footer>Footer</footer>
